flood of errors (eg OBMolRingIter is not a member of OpenBabel) with current master

hello

building from git on linux with gcc:

     ----8<----
     install:651> uname -sr
     Linux 5.0.7
     install:652> gcc --version
     gcc (GCC) 8.3.0
     Copyright (C) 2018 Free Software Foundation, Inc.
     This is free software; see the source for copying conditions.  There is NO
     warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
     ---->8----


With current master (6aff822f8d37074eebcd1901628623ea1b97f6ae) I get a
flood of errors like this:

     ----8<----
     scripts/ruby/openbabel-ruby.cpp:49397 error: 'OBMolRingIter' is not a member of 'OpenBabel'
     ---->8----

(build log is too big (~9.8MB) to attach)

My previous build (c28c7dc86db1f415c0a60abeb94497d49161ff8d) was
absolutely fine, as always.

Is there any problem or am I missing something?

thanks a lot
ciao
-g

--- a/scripts/openbabel-R.i
+++ b/scripts/openbabel-R.i
@@ -56,6 +56,9 @@
 #include <openbabel/stereo/cistrans.h>
 #include <openbabel/stereo/squareplanar.h>
 #include <openbabel/stereo/bindings.h>
+
+#include <openbabel/chains.h>
+#include <openbabel/obiter.h>
 %}
 
 

--- a/scripts/openbabel-csharp.i
+++ b/scripts/openbabel-csharp.i
@@ -668,6 +668,8 @@ using System.Runtime.InteropServices;
 #include <openbabel/rotor.h>
 #include <openbabel/rotamer.h>
 
+#include <openbabel/chains.h>
+#include <openbabel/obiter.h>
 %}
 
 //ignore some currently unsupported operators

--- a/scripts/openbabel-java.i
+++ b/scripts/openbabel-java.i
@@ -57,6 +57,9 @@
 #include <openbabel/stereo/cistrans.h>
 #include <openbabel/stereo/squareplanar.h>
 #include <openbabel/stereo/bindings.h>
+
+#include <openbabel/chains.h>
+#include <openbabel/obiter.h>
 %}
 
 #ifdef HAVE_EIGEN

--- a/scripts/openbabel-mono.i
+++ b/scripts/openbabel-mono.i
@@ -183,6 +183,8 @@ WRAP_ARRAY(double,double_array)
 #include <openbabel/kinetics.h>
 #include <openbabel/rotamer.h>
 
+#include <openbabel/chains.h>
+#include <openbabel/obiter.h>
 %}
 
 %module VecMath

--- a/scripts/openbabel-perl.i
+++ b/scripts/openbabel-perl.i
@@ -43,6 +43,8 @@
 #include <openbabel/rotor.h>
 #include <openbabel/rotamer.h>
 
+#include <openbabel/chains.h>
+#include <openbabel/obiter.h>
 %}
 
 %include "std_list.i"

--- a/scripts/openbabel-php.i
+++ b/scripts/openbabel-php.i
@@ -52,6 +52,9 @@
 #include <openbabel/isomorphism.h>
 #include <openbabel/query.h>
 #include <openbabel/canon.h>
+
+#include <openbabel/chains.h>
+#include <openbabel/obiter.h>
 %}
 
 #ifdef HAVE_EIGEN

--- a/scripts/openbabel-ruby.i
+++ b/scripts/openbabel-ruby.i
@@ -69,6 +69,8 @@
 #include <openbabel/stereo/squareplanar.h>
 #include <openbabel/stereo/bindings.h>
 
+#include <openbabel/chains.h>
+#include <openbabel/obiter.h>
 #include <openbabel/kekulize.h>
 %}
 

